pkg/cmd/admin/migrate/storage/storage.go

```diff
@@ -68,23 +68,89 @@ func NewCmdMigrateAPIStorage(name, fullName string, f *clientcmd.Factory, in io.
 
 			Include: []string{"*"},
 			DefaultExcludes: []unversioned.GroupResource{
+				// openshift resources:
 				{Resource: "appliedclusterresourcequotas"},
-				{Resource: "bindings"},
-				{Resource: "deploymentconfigrollbacks"},
-				{Resource: "events"},
 				{Resource: "imagestreamimages"}, {Resource: "imagestreamtags"}, {Resource: "imagestreammappings"}, {Resource: "imagestreamimports"},
 				{Resource: "projectrequests"}, {Resource: "projects"},
-				{Resource: "componentstatuses"},
 				{Resource: "clusterrolebindings"}, {Resource: "rolebindings"},
 				{Resource: "clusterroles"}, {Resource: "roles"},
 				{Resource: "resourceaccessreviews"}, {Resource: "localresourceaccessreviews"}, {Resource: "subjectaccessreviews"},
 				{Resource: "selfsubjectrulesreviews"}, {Resource: "localsubjectaccessreviews"},
+				{Resource: "useridentitymappings"},
+				{Resource: "podsecuritypolicyreviews"}, {Resource: "podsecuritypolicyselfsubjectreviews"}, {Resource: "podsecuritypolicysubjectreviews"},
+
+				// kubernetes resources:
+				{Resource: "bindings"},
+				{Resource: "deploymentconfigrollbacks"},
+				{Resource: "events"},
+				{Resource: "componentstatuses"},
 				{Resource: "replicationcontrollerdummies.extensions"},
 				{Resource: "podtemplates"},
-				{Resource: "useridentitymappings"},
+				{Resource: "selfsubjectaccessreviews", Group: "authorization.k8s.io"}, {Resource: "localsubjectaccessreviews", Group: "authorization.k8s.io"},
 			},
 			// Resources known to share the same storage
 			OverlappingResources: []sets.String{
+				// openshift resources:
+				sets.NewString("deploymentconfigs.apps.openshift.io", "deploymentconfigs"),
```

Comment:
Eh I feel like this should happen in some decentralized way just like registration, or at least in a way that is not so easy to have a typo.